common/src/IO/Tokenizer.cpp

```diff
@@ -104,8 +104,16 @@ namespace TrenchBroom {
         
         void TokenizerState::advance() {
             errorIfEof();
-            
+
             switch (curChar()) {
+                case '\r':
+                    if (lookAhead() == '\n') {
+                        ++m_column;
+                        break;
```

Comment:
Do you need to set `m_escaped = false;` before the break here, like the other cases do?